app/controllers/landing_page_controller.rb

```diff
@@ -20,13 +21,19 @@ def index
     cid = community_id(request)
     version = CLP::LandingPageStore.released_version(cid)
 
-    # TODO Ideally we would do the caching based upon just clp_version
-    # and avoid loading and parsing the (potentially) big structure
-    # JSON.
-    begin
-      structure = CLP::LandingPageStore.load_structure(cid, version)
+    cache_time = APP_CONFIG[:clp_cache_time] ? APP_CONFIG[:clp_cache_time].to_i : 60
```

Comment:
I think the fallback is unnecessary because defaults already specified the default value so this always exists.